single_tech_samples/synapseanalytics/sample3_integration_testing/infrastructure/modules/synapse.bicep

```diff
@@ -0,0 +1,168 @@
+param project string
+param env string
+param location string = resourceGroup().location
+param deployment_id string
+
+param synStorageAccount string = '${project}st2${env}${deployment_id}'
+param mainStorageAccount string = '${project}st${env}${deployment_id}'
```

Comment:
I think these values can be passed in directly from the main.bicep file by referencing the module output. As long as the storage account and storage file system 'name' property is outputted in the module.

Reply:
oh and also on line 9 for the keyvault name

Reply:
same comment as above. cc @devlace